src/Tabs/index.stories.js

```diff
@@ -33,6 +33,45 @@ DefaultSelectedTab.args = {
   defaultSelectedIndex: 1,
 };
 
+export const ResponsiveTabs = (args) => (
+  <div
+    style={{
+      display: "flex",
+      width: 200,
```

Comment:
adjusting the width to be closer to something like our "s" media query

```suggestion
      width: "360px",
```